extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfig.java

```diff
@@ -182,4 +189,18 @@ default boolean isAnyPropertySet() {
         }
     }
 
+    @ConfigGroup
+    interface HibernateOrmConfigDevUI {
+        /**
+         * Activate or disable the HQL console dev ui page.
+         */
+        @WithDefault("true")
+        boolean enableHqlConsole();
```

Comment:
I'm not sure what this is for... If people don't want to use it, they can just ignore it?

Reply:
Removed.